### PR TITLE
fix the case that column length is not right in CLI when the value contains Chinese characters

### PR DESCRIPTION
**Pull request by weizihan0110**
## Result

Before:
image

After:
image

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [x] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

File: cli/src/main/java/org/apache/iotdb/cli/AbstractCli.java [hunk 1]
```diff
@@ -18,10 +18,12 @@
  */
 package org.apache.iotdb.cli;
 
+import static org.apache.iotdb.cli.utils.IoTPrinter.*;
+
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
-import java.io.PrintStream;
+import java.lang.Character.UnicodeScript;
```
Comment (HTHou) [inline cli/src/main/java/org/apache/iotdb/cli/AbstractCli.java:26]: This import seems unnecessary. I think it should be remove.

**Review by HTHou — APPROVED** [review]
It looks great! 
Screen Shot 2021-01-06 at 10 08 18 AM

File: cli/src/main/java/org/apache/iotdb/cli/AbstractCli.java (hunk 1 above)
Comment (weizihan0110) [reply cli/src/main/java/org/apache/iotdb/cli/AbstractCli.java]: > This `import java.lang.Character.UnicodeScript` seems unnecessary. I think it should be removed.

Hi, already optimize imports

**Comment by weizihan0110** [comment]
running Travis CI failed, could you run it again?

**Comment by weizihan0110** [comment]
> It looks great!
> Screen Shot 2021-01-06 at 10 08 18 AM

running Travis CI failed, could you run it again?

**Comment by HTHou** [comment]
> running Travis CI failed, could you run it again?

Sure.

**Pull request merged**